apis/config/v1alpha2/configuration_types.go

```diff
@@ -77,3 +81,12 @@ type InternalCertManagement struct {
 	// Defaults to kueue-webhook-server-cert.
 	WebhookSecretName *string `json:"webhookSecretName,omitempty"`
 }
+
+type ClientConnection struct {
+	// QPS controls the number of queries per second allowed for K8S api server
+	// connection.
+	QPS *float32 `json:"qps,omitempty"`
```

Comment:
https://github.com/kubernetes/kubernetes/blob/c0c386b9c9c0eb798e8ec8d37fb0b2dd6e4554d7/staging/src/k8s.io/component-base/config/v1alpha1/types.go#L79
```suggestion
	QPS float32 `json:"qps,omitempty"`
```

`int32` below

Reply:
Everywhere else in this config we have pointers. I don't see a reason to break convention to potentially reuse a struct with 3 additional not-so-needed fields.

Reply:
Sounds good, just move from `*int` to `*int32`

Reply:
Done.